ff/src/fields/models/fp6_2over3.rs

```diff
@@ -23,6 +23,15 @@ pub trait Fp6Parameters: 'static + Send + Sync {
     }
 }
 
+impl<P: Fp6Parameters> CyclotomicField for Fp6<P> {
+    fn cyclotomic_inverse_in_place(&mut self) -> Option<&mut Self> {
```

Comment:
I think technically we can also have cyclotomic squaring ala Fp12, but I didn't want to implement that in this PR.